src/supertux/menu/options_menu.cpp

```diff
@@ -179,6 +183,77 @@ OptionsMenu::OptionsMenu(bool complete) :
     resolutions.push_back(fullscreen_size_str);
   }
 
+  // Sound Volume
+  sound_volumes.clear();
```

Comment:
In the interface. Restricting the user to arbitrarily imposed options is no good idea.